playbooks/deploy.yaml

```
@@ -51,9 +49,9 @@
       - name: candlepin
```

We now have a variable for this:
```suggestion
      - name: "{{ candlepin_database_user }}"
```